lib/dateTime.js

```diff
@@ -1128,10 +1128,24 @@ export function formatDateTime(date, options) {
 
 }
 
-export function formatDateTimeFromTimestamp(timestamp, options) {
+function parseLocalDateTimeFromTimestamp(timestamp, options) {
 	const utcDate = new Date(timestamp);
 	const local = convertJsDateToLocalDateTime(utcDate);
 	if (!local) return formatDateTime(utcDate, options);
```

Comment:
I think you'd want this to return just `utcDate` in this case, since `formatDateTime` is then applied to whatever is returned.

Reply:
Ah. Good point. I've made that change now.